### DISCUSSION TITLE
It can be interresing honestly:

### DISCUSSION THREAD

**Discussion by thedoggaming16** [helge17/tuxguitar#98] [General]
https://sourceforge.net/p/tuxguitar/feature-requests/48/

**Comment by guiv42** [comment]
I don't really understand: what's the point of changing only the *tuner* frequency? You would lose the ability to practice while TuxGuitar is playing.
In this case, why not use a dedicated tuner tool (hardware or software) rather than TuxGuitar? TuxGuitar is essentially a tablature editor, and tuner is just an add-on (I don't consider it a core feature).

Changing frequencies of both *tuner* and *player* would seem more logical to me, but in terms of implementation it's not the same complexity
